script.js

```diff
@@ -90,12 +99,30 @@ function showModal(winner) {
   modal.style.display = "block";
   if (winner.includes("1")) {
     displayWinner.style.color = "#f9b233";
+    playWinAudio();
   }
   if (winner.includes("2")) {
     displayWinner.style.color = "#e6332a";
+    playWinAudio();
+  }
+
+  if (winner.includes("draw")) {
+    displayWinner.style.color = "#75aa31";
+    playDrawAudio();
   }
-  displayWinner.innerHTML = winner + '<p class="newGameMessage"> Click Start For a New Game!</p>';
+  displayWinner.innerHTML =
+    winner + '<p class="newGameMessage"> Click Start For a New Game!</p>';
```

Comment:
Can you remove the space before Click?
Might make it go onto one line? But hopefully not